analyse.py

```diff
@@ -298,12 +296,196 @@ def get_file_messages(fp):
 
 
 def get_folder_messages(input_dir):
-    files = [f for f in os.listdir(input_dir) if f.endswith(".json")]
+    files = [f for f in os.listdir(input_dir) if f.endswith(".json") and f.startswith("message_")]
     messages = []
     for f in files:
         messages.extend(get_file_messages(os.path.join(input_dir, f)))
     return messages
 
+def get_participants(input_dir):
+    files = [f for f in os.listdir(input_dir) if f.endswith(".json") and f.startswith("message_")]

```

Comment:
Could we extract this to a function if we're going to reuse the same code as `get_folder_messages`?